blocks/calendar-events/calendar-events.css

```diff
@@ -0,0 +1,76 @@
+.calendar-events.block {
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: auto;
+    justify-items: center;
+    gap: 28px;
+    margin: 0 auto;
+}
+
+.calendar-events.block > div {
+    display: grid;
+    grid-template-columns: 80px auto;
+    padding: 20px;
+    border: solid 1px transparent;
+    border-radius: 10px;
+    box-shadow: 0 4px 4px #0003;
+    width: 363px;
+}
+
+.calendar-events.block > div > div:first-of-type {
+    background: var(--red);
+    height: 64px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    margin: 0 14px 0 0;
+    border: solid 1px transparent;
+}
+
+.calendar-events.block > div > div:first-of-type > h3,
+.calendar-events.block > div > div:first-of-type > p {
+    color: var(--white);
+    line-height: 22px;
+    margin: 0;
+}
+
+.calendar-events.block > div > div:first-of-type > h3 {
+    font-size: var(--body-font-size-xxl);
+}
+
+.calendar-events.block > div > div:first-of-type > p {
+    font-size: var(--body-font-size-s);
+}
+
+.calendar-events.block > div > div:nth-of-type(2) {
+    font-family: var(--body-font-family);
+    line-height: 22px;
+}
+
+.calendar-events.block > div > div:nth-of-type(2) > p {
+    font-size: var(--body-font-size-l);
+}
+
+@media screen and (max-width: 899px){
+    .calendar-events.block > div {
+        width: 100%;
```

Comment:
@bstopp if you would like this to be an explicit width instead of relying on the page content/layout width, just let me know and I'll change it.